e2e/test-static-pod.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -o errexit
+
+echo "Creating network attachment definition"
+kubectl create -f static-pod-nad.yml
+
+echo "Creating static pod config file"
+docker cp simple-static-pod.yml multus-whereabouts-worker:/etc/kubernetes/manifests/static-web.yaml
```

Comment:
`multus-whereabouts-worker` is not correct container name of multus e2e and that's why CI is failed.
Could you please check it?